transport/src/iosMain/kotlin/com/genesys/cloud/messenger/transport/network/PlatformSocket.kt

```diff
@@ -71,7 +72,7 @@ internal actual class PlatformSocket actual constructor(
                     reason: NSData?,
                 ) {
                     val why = reason?.string() ?: "Reason not specified."
-                    log.i { "Socket did close (code: $didCloseWithCode, reason: $why). Active: $active." }
+                    log.i { LogMessages.socketDidClose(didCloseWithCode.toInt(), why, active) }
```

Comment:
you can set the socketDidClose log message to accept Long as a parameter.
This will get rid of toInt transformation. Also, what will happen if the `didCloseWithCode` is long that to big to be cast to int?